devlab/tests/test_resource_migration.py

```diff
@@ -1,11 +1,12 @@
-import re
-import pprint
 import config
-import unittest
-import subprocess
 import functional_test
 
-from time import sleep
+import pprint
```

Comment:
It is used for error message printing:
https://github.com/MirantisWorkloadMobility/CloudFerry/blob/devel/devlab/tests/test_resource_migration.py#L431